content/en/sensitive_data_scanner/guide/best_practices_for_creating_custom_rules.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Best Practices for Creating Custom Rules
+disable_toc: false
+further_reading:
+- link: "/sensitive_data_scanner/"
+  tag: "Documentation"
+  text: "Set up Sensitive Data Scanner"
+- link: "/sensitive_data_scanner/regular_expression_syntax"
+  tag: "Documentation"
+  text: "Regular expression syntax for custom rules"
+---
+
+## Overview
+
+Sensitive Data Scanner uses scanning rules to identify, tag, and optionally redact sensitive data in your logs, APM events, and RUM events. Use [out-of-the-box scanning rules][3] or create your own custom rules using [regular expression][1] (regex) patterns. This guide goes over best practices for creating custom rules with regex.
+
+## Use precise regex patterns
+
+Define regex patterns that are as precise as possible because generic patterns result in more false positives. To refine your regex pattern, add test data in the sample data tester when you are creating a custom rule. See step 2 in [Add a custom scanning rule][2] for more information.
+
+{{< img src="sensitive_data_scanner/guides/regex_sample_test.mp4" alt="Testing a regex pattern with a sample that matches and one that does not" video=true >}}
+
+## Refine regex pattern matching
+
+Add keywords to the keyword dictionary to refine detection accuracy for matching regex patterns. For example, if you are scanning for passwords, you can add keywords like `password`, `token`, `secret`, and `credential`. You can also require that these keywords be within a specified number of characters of a match. By default, keywords must be within 30 characters before a matched value. See step 2 in [Add a custom scanning rule][2] for more information.
+
+{{< img src="sensitive_data_scanner/guides/password_keyword.png" alt="A keyword dictionary with password, token, secret, credential" style="width:90%;" >}}
+
+To make matches more precise, you can also do one of the following:
+
+- Scan the entire event but exclude certain attributes from getting scanned. For example, if you are scanning for personally identifiable information (PII) like names, you might want to exclude attributes such as `resource_name` and `namespace`.
+- Scan for specific attributes to limit the scope of the data that is scanned. For example, if you are scanning for names, you can choose specific attributes such as `first_name` and `last_name`.
```

Comment:
```suggestion
- Scan for specific attributes to narrow the scope of the data that is scanned. For example, if you are scanning for names, you can target attributes such as `first_name` and `last_name`.
```

Reply:
PM had requested to say "choose specific attributes", so am going to leave that.